packages/website/src/components/Announcement.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Text from './Text';
+import { H4 } from './Heading';
+import Box from './Box';
+import Section from './Section';
+
+interface AnnouncementProps {
+  location: Object;
+}
+
+const AnnouncementBox = styled(Box)`
+  background: #ffe9bf;
+  width: 60%;
+  padding: ${({ theme }) => theme.space.medium} ${({ theme }) => theme.space.large};
+  margin-bottom: ${({ theme }) => theme.space.medium};
+  border-radius: 4px;
+`;
+
+const redirected = (props): boolean => {
+  const params = new URLSearchParams(props.search);
+  const from_old_website: string = params.get('from-old-website')!;
+  return from_old_website == 'true' ? true : false;
+};
+
+const from_where = props => {
+  const params = new URLSearchParams(props.search);
+  const from_which_page: string = params.get('archived-page')!;
```

Comment:
We'd need to prepend the archive site url to the param (like `https://v1.bigtest.io`). It could be argued that we should store that URL in some variable at the project-level, but this announcement is a very unique thing so I think it's fine to just hard-code it here.

Reply:
okay will do. and taras helped me with ruby last night to make this much easier.

Reply:
i prepended the url from the old website so it's not necessary to hard-coded it from the new website.